### ISSUE TITLE
Add graphs to visualize training

### ISSUE THREAD

**Issue by LBerth** [Caterpillar314/paac#6]
We need to add a new graph to tensorboard to visualize the loss function, the rewards, the episode length, etc... With it, it will be easier to monitor the algorithm.

**Comment by LBerth** [comment]
For each graph, the x axis is in millions of steps (=actions/decisions).

**Monitor the rewards**
We plotted the max, min, mean, standard deviation and standard deviation divided by the mean calculated over the score of the last 50 episodes.
![image](https://user-images.githubusercontent.com/19271957/30904172-7e0a1082-a371-11e7-992d-dd7f8fd4c08f.png)

**Monitor the episode length**
We plotted the max, min, mean, standard deviation and standard deviation divided by the mean calculated over the episode length (in number of actions) of the last 50 episodes.
![image](https://user-images.githubusercontent.com/19271957/30904314-ed461838-a371-11e7-9a40-e145ac488898.png)

**Monitor the loss**
We plotted the advantage of the actor, the loss of the actor, the loss of the critic and the global loss.
![image](https://user-images.githubusercontent.com/19271957/30904459-56f4619a-a372-11e7-8d7a-b1c23cf9b508.png)

**Monitor the parameters**
For now, we only plotted the evolution of the learning rate.
![image](https://user-images.githubusercontent.com/19271957/30904574-a2e0fba4-a372-11e7-9213-91ae991035fb.png)

**Visualize the output of Conv Layers**
You can see the different features for each Conv Layer : 
![image](https://user-images.githubusercontent.com/19271957/33275177-7b34f5ba-d392-11e7-9642-6958d66b90c8.png)

**Monitor the gradients**
![image](https://user-images.githubusercontent.com/19271957/33275274-b40826d2-d392-11e7-9bab-0bff6b56f2cf.png)

**Visualize the Tensorflow graph**
![image](https://user-images.githubusercontent.com/19271957/33275355-f06384a0-d392-11e7-8fd7-ae2e2b3cb2a1.png)

**Comment by LBerth** [comment]
Comment and uncomment the summaries in paac.py and actor_learner.py to choose the graphs you want.

Be careful : using many graphs, and especially Conv layer outputs and gradients histograms , will use a lot of disk space ! (up to 40 GB for a 80 million steps training)
